utils/rgmercs_utils.lua

```diff
@@ -861,12 +861,16 @@ function Utils.UseSong(songName, targetId, bAllowMem)
             mq.doevents()
             mq.delay(1)
         end
-
+        
         if targetId == me.ID() then
-            mq.delay("5s", function() return me.Song(songName).Duration.Seconds() > oldDuration end)
+            mq.delay("3s", function() return me.Song(songName).Duration.Seconds() > oldDuration end)

```

Comment:
any reason not to leave this at 5s? It will early out anyway if duration > old duration

Reply:
It was not early-outing in my tests, so the bard would stand around a few seconds doing nothing

Reply:
would be worth looking at what the value of old duration and Duration.Seconds() is and figure out why it isn't working but we can do that later.